<launch>

    <arg name="rate" default="100.0"/>
    <param name="log" type="bool" value="true"/>

    <node pkg="contacts_opt" type="ifopt_contacts_node" name="ifopt_contacts_node" required="true" output="screen">
	<param name="is_model_floating_base" type="bool" value="true"/>
        <param name="model_type" value="RBDL"/>
        <param name="rate" value="$(arg rate)"/>    
    </node> 

    
</launch>
